content/docs/user-guide/machine-learning-pipelines/index.md

```diff
@@ -0,0 +1,18 @@
+# Machine Learning Pipelines
```

Comment:
Why not just `Pipelines`? We already have a lot of content focused on ML, but I worry we are pushing too hard and alienating non-ML users when the content has little that is specific to ML.